src/components/Comments/CommentForm.js

```diff
@@ -21,6 +21,7 @@ class CommentForm extends React.Component {
     onImageInserted: PropTypes.func,
     onImageInvalid: PropTypes.func,
     onSubmit: PropTypes.func,
+    commentSubmitted: PropTypes.bool,
```

Comment:
We usually have `func` props at the bottom. Can you move this prop up? I think it makes sense to rename this props to `submitted`.